docs/guides/documentation/releases/README.md

```diff
@@ -0,0 +1,94 @@
+# Releases
+
+<pre align="center">Streamline software releases through consistent templates, automated aggregation and notifications.</pre>
```

Comment:
Why is this in fixed-width/centered? Is this an AMMOS standard way of identifying a document's scope? 🤔

Reply:
That's what I've seen as the template in other places.

Reply:
Same as above comments, straight from [our SLIM Guide Template](https://nasa-ammos.github.io/slim/docs/contribute/submit-best-practice#use-our-standard-guide-template).

I think the intention of the `<pre align="center">` is to ensure the text within sticks out and is kept short. Though I've noticed it doesn't compact well on small screens. Once again, if you have suggestions to the template happy to hear about them!